common/src/main/java/tachyon/util/CommonUtils.java

```diff
@@ -66,7 +96,7 @@ public static void sleepMs(Logger logger, long timeMs, boolean shouldInterrupt)
   }
 
   /**
-   * Common empty loop utilities to warm up
+   * Common empty loop utilities to warm up.
    */
   public static void warmUpLoop() {
     for (int k = 0; k < 10000000; k ++) {
```

Comment:
I think it depends on the compiler optimizations. This example https://ideone.com/SLfjnz suggests that the empty loop is not always optimized away.

The function is currently used in one place in the code base. I'll chat with @calvinjia to see if we want to keep it.